python/cuml/cuml/tests/test_simpl_set.py

```diff
@@ -141,50 +145,52 @@ def test_simplicial_set_embedding(
     X = X.get()
 
     ref_fss_graph = ref_fuzzy_simplicial_set(
-        X, n_neighbors, random_state, metric
+        X, n_neighbors=n_neighbors, random_state=random_state, metric=metric
     )[0]
     ref_embedding = ref_simplicial_set_embedding(
         X,
-        ref_fss_graph,
-        n_components,
-        initial_alpha,
-        a,
-        b,
-        gamma,
-        negative_sample_rate,
-        n_epochs,
-        init,
-        np.random.RandomState(random_state),
-        dist.named_distances_with_gradients[metric],
-        metric_kwds,
-        densmap,
-        densmap_kwds,
-        output_dens,
+        graph=ref_fss_graph,
+        n_components=n_components,
+        initial_alpha=initial_alpha,
+        a=a,
+        b=b,
+        gamma=gamma,
+        negative_sample_rate=negative_sample_rate,
+        n_epochs=n_epochs,
+        init=init,
+        random_state=np.random.RandomState(random_state),
+        metric=metric,
+        metric_kwds=metric_kwds,
+        densmap=densmap,
+        densmap_kwds=densmap_kwds,
+        output_dens=output_dens,
```

Comment:
I assume this is part of the general clean-up rather than a functional difference?